### ISSUE TITLE
Minor issue: comments cut off

### ISSUE THREAD

**Issue by AngledLuffa** [Orange-OpenSource/conllueditor#47]
I find that occasionally comments get cut off depending on the format:

![Image](https://github.com/user-attachments/assets/1665d568-06aa-432e-8944-0e785e479ce5)

The original comment in this image was 

```
# semgrex pattern |{cpos:ADP} <det {}| matched at 29:جا
```

**Comment by AngledLuffa** [comment]
I don't believe the conllu specification disallows < in comments, although I could be wrong

**Comment by jheinecke** [comment]
Thanks for bringing my attention to this. This was a display issue, the < and > where interpreted as (incomplete) HTML tags. I corrected this in V2.29.2
